docs/guide/snapshot.md

```diff
@@ -191,3 +191,26 @@ export default defineConfig({
   }
 })
 ```
+
+#### 3. Chevron `>` is used as a separator instead of colon `:` for hints
+
+Vitest uses chevron `>` as a separator instead of colon `:` for readability.
```

Comment:
```suggestion
#### 3. Chevron `>` is used as a separator instead of colon `:` for custom messages

Vitest uses chevron `>` as a separator instead of colon `:` for readability, when a custom message is passed during creation of a snapshot file.
```